CollectNewspaperKarel.java

```diff
@@ -11,7 +11,25 @@
 import stanford.karel.*;
 
 public class CollectNewspaperKarel extends SuperKarel {
+c void run(){
```

Comment:
This line has a c in the beginning of it and because of that this results into a syntax error and the program can't run.